modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/config/LoaderConfig.scala

```diff
@@ -127,6 +129,65 @@ object LoaderConfig {
       }
     }
 
+    case class LocalFS(path: PathInfo) extends Source
+    object LocalFS {
+      /**
+        * Contains information about the given path
+        * @param path Path which contains events
+        * @param pathType Specifies whether given path is absolute or relative
+        */
+      case class PathInfo(path: Path, pathType: PathType) {
```

Comment:
I guess we can do that. Will look into it 👍

Reply:
For this and `fileExists` check, I guess we can implement them as you suggest but I am not sure config parser is right place for IO action. Do you have any idea in your mind how we can introduce `IO` in config parsing part ?

Reply:
I created `PathInfo` in order to wrap all path related fields. Currently, `LocalFS` has only `PathInfo` field but it might have additional field later on. IMHO, it is better like that.